packages/amazonq/src/lsp/chat/webviewProvider.ts

```diff
@@ -34,13 +34,14 @@ export class AmazonQChatViewProvider implements WebviewViewProvider {
         }
 
         const uiPath = webviewView.webview.asWebviewUri(Uri.parse(this.mynahUIPath)).toString()
-        webviewView.webview.html = getWebviewContent(uiPath)
+        const disclaimerAcknowledged = globals.globalState.tryGet('aws.amazonq.disclaimerAcknowledged', Boolean, false)
```

Comment:
Should this not be stored in `suppressPrompts` ? https://github.com/aws/aws-toolkit-vscode/blob/b1fae85369d794172d730cc576fe27a41b6c1778/packages/core/src/shared/settings.ts#L658

Reply:
this is just porting the existing implementation to the lsp one. Maybe originally yes, but now that's going to require all sorts of extra re-working to port the global state to suppress prompts

Reply:
No, its mostly that. It's just going to be annoying for users. I've set it up so that if they had previous acknowledged it it moves it to supress prompts instead